src/assets/scripts/client/canvas/CanvasController.js

```diff
@@ -603,21 +622,19 @@ export default class CanvasController {
         cc.lineJoin = 'round';
         cc.font = BASE_CANVAS_FONT;
 
-        _forEach(this._navigationLibrary.realFixes, (fix, i) => {
+        // for (const fix in this._navigationLibrary.realFixes) {
+        const fixes = this._navigationLibrary.realFixes;
+        for (let i = 0; i < fixes.length; i++) {
```

Comment:
we can still use `fix` here, and we should be, but we have to do it manually:
```javascript
for (let i = 0; i < fixes.length; i++) {
  const fix = fixes[i];
}
```
this also gives us a `const` because each iteration through the loop is a new context (thus, a new `const`).